site/src/pages/UsersPage/UsersTable/UsersTableBody.tsx

```diff
@@ -136,125 +136,91 @@ export const UsersTableBody: FC<
           </Cond>
         </ChooseOne>
       </Cond>
+
       <Cond>
-        <>
-          {users &&
-            users.map((user) => {
-              // When the user has no role we want to show they are a Member
-              const fallbackRole: TypesGen.Role = {
-                name: "member",
-                display_name: "Member",
-              };
-              const userRoles =
-                user.roles.length === 0
-                  ? [fallbackRole]
-                  : sortRoles(user.roles);
+        {users?.map((user) => (
```

Comment:
Not too much changed here, but because the indentation level changed, it might help to view this file with whitespace changes off